schema/SCHEMA_QUDT-v2.1.ttl

```diff
@@ -2941,7 +2941,7 @@ qudt:ucumCaseSensitiveCode
   owl:deprecated true ;
 .
 qudt:ucumCode
-  a owl:AnnotationProperty ;
+  a owl:ObjectProperty ;
```

Comment:
qudt:UCUMcs is a Datatype, so I think this should be a DatatypeProperty.

Reply:
+1

Reply:
Yes, qudt:ucumCode shoudl be a owl:DatatypeProperty. there is a datatype specified, and under UCUM rules the code is semantically meaningful, not just an annotation. In particular, the relationship should be InverseFunctional, though we can't express this in OWL DL because owl:InverseFunctionalProperty is not available for Datatype properties.

Reply:
fixed on my branch